src/ReleaseHistory.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* BUGFIX: Fix `JsonSerializationException` when reading Sarif v1.0.0 when telemetry is enabled. [#535](https://github.com/microsoft/binskim/pull/535)
```

Comment:
>Fix [](http://example.com/codeflow?start=10&length=3)

Fix `JsonSerializationExxception' that occurs when saving SARIF v1 with telemetry enabled.

Reply:
updated!
thanks for the feedback!